docs_src/microservices/device/remote/Ch-RemoteNonSecure.md

```diff
@@ -0,0 +1,134 @@
+# Non-Secure Mode
+
+## Remote deployment of device services in non-secure mode
+Usually microservices are deployed using multiple nodes for scalability, availability and performance 
+instead of running all services in one node.
+
+This page provides an example of remote deployment of `device-usb-camera` service using multiple nodes in non-secure mode.
+The deployment can be done by running the service `natively` or by running it in `docker`.
+
+## Example
+This example uses 2 nodes for remote deployment. One node (remote) is used to run all Edgex core services in docker and the other node (host)
```

Comment:
this looks confusing to me. which node is host? the one running edgex or device usb?
can we say some thing like this? This example uses 2 nodes for remote deployment. One node (gateway) is used to run all EdgeX core services in Docker and the other node (remote) is used to run USB device service.